spark/v3.3/spark/src/main/java/org/apache/iceberg/spark/source/SparkWrite.java

```diff
@@ -202,6 +202,11 @@ private void commitOperation(SnapshotUpdate<?> operation, String description) {
       operation.stageOnly();
     }
 
+    String branch = writeConf.branch();
```

Comment:
We init other vars like `wapEnabled` and `wapId` earlier.
Would it make sense to follow that for `branch` too?

Reply:
good point, I did it for `SparkPositionDeltaWrite`, did not do it for `SparkWrite`